src/GafferTestModule/LRUCacheTest.cpp

```diff
@@ -485,6 +487,79 @@ void testLRUCacheExceptions( const std::string &policy )
 	DispatchTest<TestLRUCacheExceptions>()( policy );
 }
 
+template<template<typename> class Policy>
+struct TestLRUCacheCancellation
+{
+
+	void operator()()
+	{
+		std::vector<int> calls;
+		IECore::Canceller *canceller = new IECore::Canceller();
```

Comment:
Prefer `std::unique_ptr` and `boost::make_unique` over `new` and `delete`.

Reply:
Sorry should have seen those elsewhere in that file... fixed in https://github.com/GafferHQ/gaffer/pull/3474/commits/0301b8544cf37a250b00a7d79f88d89c0431078f.